fe/fe-core/src/main/java/org/apache/doris/nereids/operators/plans/logical/LogicalAggregate.java

```diff
@@ -56,6 +57,16 @@ public LogicalAggregate(List<Expression> groupByExprList, List<NamedExpression>
         super(OperatorType.LOGICAL_AGGREGATION);
         this.groupByExprList = groupByExprList;
         this.outputExpressionList = outputExpressionList;
+        this.disassembled = false;
```

Comment:
No need this line. The default value of disassembled is false

Reply:
It's a `final` field, so we must init it explicitly in the constructor